CONTRIBUTING.md

```diff
@@ -134,13 +134,13 @@ Must be one of the following:
 
 ### Scope
 The scope should be the name of the npm package affected (as perceived by the person reading the changelog 
-generated from commit messages).
+generated from commit messages). Capitalize the first letter.
 
 The following is the list of supported scopes:
 
-* **core**
-* **platform**
-* **doc-app**
+* **Core**
+* **Platform**
+* **Doc-app**
```

Comment:
@InnaAtanasova @droshev when and where do we use `Doc-app`?  I have mostly only seen `docs` type used, with `Core` or `Platform` scope.

Reply:
renamed `doc-app` to `Docs`